### DISCUSSION TITLE
Display gameplay cursor while changing cursor size

### DISCUSSION THREAD

**Discussion by MulaHwaka** [ppy/osu#29343] [Ideas]
At the moment when you are changing your gamplay cursor size you cannot see how big/small you are making it until you load into a map as it is not displayed. A change to the ui to allow the user to see the size of the cursor they are adjusting would be appreciated.

**Comment by peppy** [comment]
We have a lot of work to do on the usability of settings. There are a lot of other settings which would benefit from a local widget including preview / otherwise.
